internal/services/postgres/postgresql_flexible_server_resource.go

```diff
@@ -606,6 +614,13 @@ func expandArmServerBackup(d *pluginsdk.ResourceData) *postgresqlflexibleservers
 		backup.BackupRetentionDays = utils.Int32(int32(v.(int)))
 	}
 
+	geoRedundantBackupEnabled := d.Get("geo_redundant_backup_enabled").(bool)
+	if geoRedundantBackupEnabled {
```

Comment:
these lines can be combined?
```suggestion
	if geoRedundantBackupEnabled := d.Get("geo_redundant_backup_enabled").(bool); geoRedundantBackupEnabled {
```